### PR TITLE
Add Python Multiprocessing Helpers

### PR DESCRIPTION
**Pull request by CCInc**

### CODE REVIEW

**Review by leo-stan — COMMENTED** [review]
Well done! Could we add a bunch more comments in the code? I feel like future us will be happy when we have to dive back into them later.

File: example/example_mp.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    rotated_points += rotation_center
+
+    points.x = rotated_points[:, 0]
+    points.y = rotated_points[:, 1]
+    points.z = rotated_points[:, 2]
+
+    return points
+
+
+def rotate_file_mp():
```
[the remark this answers is not in the document]
Comment (CCInc) [reply example/example_mp.py:40]: Let me know if that's better!

File: test/utils.py [hunk 2]
```diff
@@ -10,3 +12,100 @@ def get_autzen_file():
 
 def get_data_dir():
     return DATADIRECTORY
+
+
+def generate_test_file():
```
Comment (leo-stan) [inline test/utils.py:17]: What does this do? Generate a file with random points within the bounds below? Might want to add a docstring

File: test/utils.py [hunk 3]
```diff
@@ -10,3 +12,107 @@ def get_autzen_file():
...
+
+        # Create a Points object based on the LAS header
+        points = copc.Points(las_header)
+
+        # Populate the points
+        for i in range(number_points):
+            # Create a point with a given point format
+            point = points.CreatePoint()
+            # point has getters/setters for all attributes
+            point.X = random.randint(
```
Comment (leo-stan) [inline test/utils.py:60]: This will need to get updated once #119 is merged

**Pull request merged**